src/modules/sharing/sharing.service.ts

```diff
@@ -1212,6 +1214,23 @@ export class SharingService {
         ? acceptInviteDto.encryptionKey
         : invite.encryptionKey,
     });
+
+    const ownerSharing = await this.sharingRepository.countSharing({
```

Comment:
Use a findOne from the repository, instead of a count. Also, if you are creating a sharing that is shared with the owner itself, the `sharedWith` should be the `owner.uuid`. The user with the uuid 0000... is a special Drive user which also is used as a reference for public sharing (which is not the case here, as you are accepting an invitation = private sharing).

Reply:
I'm using the `count` method because is more efficient and has better performance, why should I use the `findOne` method? 

 About the `sharedWith` I'll make the change.